medusa/tv.py

```diff
@@ -1839,33 +1840,10 @@ def get_overview(self, ep_status):
 
         if ep_status == WANTED:
             return Overview.WANTED
-        elif ep_status in (UNAIRED, UNKNOWN):
```

Comment:
@ratoaq2 backlogOverview never showed unaired|unknown. It only shows wanted and low quality